dlme_airflow/services/harvest_dag_generator.py

```diff
@@ -40,52 +44,67 @@ def default_dag_args():
     }
 
 
-def create_dag(provider, default_args) -> DAG:
```

Comment:
`create_dag()` was changed to `create_dags()` since it now can return more than one dag if the provider has `separate_dags` set to true.